Pages/Card/card.css

```diff
@@ -0,0 +1,216 @@
+  width: 20px;
+  height: 20px;
+  margin-right: 1.3rem;
+}
+
+.lower div {
+  margin-left: 0.5rem;
+}
+
+.lower small {
+  font-size: 14px;
+  color: rgb(98, 0, 238);
+  letter-spacing: 1.25px;
+  font-family: "roboto";
+  font-weight: bold;
+}
+
+.lower button {
+  border: none;
+  background-color: rgba(0, 0, 0, 0);
+  padding: 0.5rem 0.6rem;
+}
+
+.lower button:hover {
+  cursor: pointer;
+  background-color: rgba(98, 0, 238, 0.03);
+  border-radius: 5px;
+}
+
+.lower img:hover {
+  cursor: pointer;
+}
+
+/* basic-card */
+
+.basic-card-wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 50vh;
+}
+
+.basic-card .upperSection:hover {
+  background-color: #f5f5f5;
+  cursor: pointer;
+}
+
+.basic-card {
+  display: flex;
+  flex-flow: column nowrap;
+  width: 350px;
+  box-shadow: 3px 3px 80px 1px rgba(207, 207, 207, 0.5);
+  border-radius: 6px;
+}
+
+.basic-card .upperSection img {
+  width: 350px;
+  height: 196.88px;
+}
+
+.basic-card .upperContent {
+  padding: 1rem;
+}
+
+.details p {
+  color: rgba(0, 0, 0, 0.54);
+  letter-spacing: 0.25px;
+  font-size: 14px;
+}
+
+.details {
+  margin-top: 3px;
+  padding: 0 1rem;
+  padding-bottom: 0.5rem;
+  margin-bottom: 6px;
+}
+
+/* Horizontal Card */
+
+.horizontal-card-wrapper {
+  height: 50vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.horizontal-card {
+  background-color: #ffffff;
+  width: 350px;
+  height: 162px;
+  box-shadow: 3px 4px 6px rgba(207, 207, 207, 0.5);
+  border-radius: 5px;
+}
+
+.horizontal-card .upperSection {
+  display: flex;
+}
+
+.horizontal-card .upperContent {
+  margin-left: 0.9rem;
+}
+
+.horizontal-card .upperSection:hover {
+  background-color: #f5f5f5;
+  cursor: pointer;
+}
+
+.horizontal-card .upperSection img {
+  width: 110px;
+  height: 110px;
+  border-top-left-radius: 4px;
+  border-top-right-radius: 4px;
+}
+
+.horizontal-card .upperContent {
+  padding-top: 1rem;
+}
+
+.horizontal-card .lower {
+  margin-top: 0.7rem;
+}
+
+/* text-overlay */
+
+.text-overlay .upperSection {
+  position: relative;
+  /* border: red solid; */
+}
+
+.text-overlay .upperContent {
+  position: absolute;
+  left: 0;
+  color: white;
+  bottom: 3rem;
+}
+.text-overlay .upperContent h3 {
+  color: white;
+}
+
+/* Dismmisable card */
+
+.dismiss-card {
+  position: relative;
+}
+
+.dismiss-card i {
+  position: absolute;
+  top: 0.2rem;
+  right: 0.1rem;
+  padding: 0.1rem 0.25rem;
+  background: white;
+  border-radius: 50%;
+}
+
+/* card with badge */
+
+.badge-card {
+  position: relative;
+}
+
+/* The badge component will be used here will import it in dev branch and color varibles for color*/
+
+.badge-card .badge {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  top: -4px;
+  right: -9px;
+  position: absolute;
+  font-size: 0.5rem;
+  color: white;
+  font-weight: 900;
+  height: 35px;
+  width: 35px;
+  border: none;
+  background-color: red;
+  border-radius: 50%;
```

Comment:
As there are multiple properties in this class you could sort them in alphabetical order as that helps in debugging as well as code maintainability and code readability.